elements/pl-multiple-choice/pl-multiple-choice.css

```
@@ -7,3 +7,12 @@
     min-width: 1.25em;
     overflow-wrap: normal;
 }
+
+.pl-multiple-choice-hint {
+    border-left: .25rem solid #999;
+    background-color: #F9F9F9;
```

```suggestion
    background-color: rgba(0,0,0,0.05);
```
This is a suggestion, feel free to adjust if needed. The rationale for using opacity instead of color is that, if the background is set to another color (e.g., the instructor changed it to something else with CSS), this will still make the hint slightly darker than the background. A similar change may be a good consideration for the border as well.

---

Good suggestion, I changed both the border and background to use this instead.